src/ncp/ncp_base.cpp

```diff
@@ -41,14 +41,25 @@
 #include <platform/radio.h>
 #include <platform/misc.h>
 
-namespace Thread {
+namespace Thread
+{
 
 extern ThreadNetif *sThreadNetif;
 
 static NcpBase *sNcpContext = NULL;
 
 #define NCP_PLAT_RESET_REASON        (1<<31)
 
+enum
+{
+    kThreadModeTLV_Receiver    = (1 << 3),
+    kThreadModeTLV_Secure      = (1 << 2),
+    kThreadModeTLV_DeviceType  = (1 << 1),
+    kThreadModeTLV_NetworkData = (1 << 0),
+};
+
+#define UNDEFINED_RSSI      (-128)
```

Comment:
I was trying to avoid having to deal with a flag, so I picked what I figured would be an unlikely RSSI value to use as the "don't override the RSSI" indication. This behavior is only used for the whitelist, and obviously needs to be documented.

After thinking about it more, a value of 127dBm (0x7F) would likely be a better choice for this indication, since it corresponds to a *ridiculously* large power (**5 gigawatts!**).